radicle-httpd/src/api/v1/delegates.rs

```diff
@@ -0,0 +1,70 @@
+use axum::response::IntoResponse;
+use axum::routing::get;
+use axum::{Extension, Json, Router};
+use serde::Serialize;
+
+use radicle::cob::store::Store;
+use radicle::git::Oid;
+use radicle::identity::project::Payload;
+use radicle::identity::{Did, Doc};
+use radicle::storage::{ReadRepository, WriteStorage};
+
+use crate::api::axum_extra::Path;
+use crate::api::error::Error;
+use crate::api::Context;
+
+/// Project info.
+#[derive(Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Info {
+    /// Project metadata.
+    #[serde(flatten)]
+    pub payload: Payload,
+    pub head: Oid,
+    pub patches: usize,
+    pub issues: usize,
+}
+
+pub fn router(ctx: Context) -> Router {
+    Router::new()
+        .route(
+            "/delegates/:delegate/projects",
+            get(delegates_projects_handler),
+        )
+        .layer(Extension(ctx))
+}
+
+/// List all projects which delegate is a part of.
+/// `GET /delegates/:delegate/projects`
+async fn delegates_projects_handler(
+    Extension(ctx): Extension<Context>,
+    Path(delegate): Path<Did>,
+) -> impl IntoResponse {
+    let storage = &ctx.profile.storage;
+    let projects = storage
+        .projects()?
+        .into_iter()
+        .filter_map(|id| {
+            let Ok(repo) = storage.repository(id) else { return None };
+            let Ok((_, head)) = repo.head() else { return None };
+            let Ok(Doc { payload, delegates, .. }) = repo.project_of(ctx.profile.id()) else { return None };
+
+            if !delegates.iter().any(|d| d.id == delegate) {
+                return None;
+            }
+
+            let Ok(cobs) = Store::open(ctx.profile.public_key, &repo) else { return None };
+            let Ok(issues) = cobs.issues().count() else { return None };
+            let Ok(patches) = cobs.patches().count() else { return None };
+
```

Comment:
like before, we're not interrupting the report in case of error and simply skipping problematic items. we could also have defaults at this stage, e.g. return `issues: 0, patches: 0` in case of error.

we can also use a regular loop and return error to api client, which I'm not a fan of. rather keep error loggings on server side.